src/TLuaInterpreter.cpp

```diff
@@ -309,6 +309,11 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString(lua_tostring( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
         }
+        else if( lua_isboolean( L, i ) )
```

Comment:
lua_isboolean(...) returns 1 only if the value is of type boolean, and lua_isstring(...) and lua_isnumber(...) both return 0 for booleans, so I don't think the order should matter. Anything that satisfies lua_isboolean() will not satisfy lua_isstring() or lua_isnumber() and vice versa. Unless I'm misinterpreting some docs.